python/sglang/srt/layers/attention/double_sparsity_backend.py

```diff
@@ -165,7 +165,13 @@ def get_cuda_graph_seq_len_fill_value(self):
         return 1
 
     def forward_extend(
-        self, q, k, v, layer: RadixAttention, forward_batch: ForwardBatch
+        self,
```

Comment:
It would be better to add a compatibility test for double sparsity. cc @ispobock @andy-yang-1

Reply:
There is an [unit test](https://github.com/sgl-project/sglang/blob/main/test/srt/test_double_sparsity.py) for double sparsity and it passed.

Reply:
ok